src/spaceship.cpp

```diff
@@ -89,6 +89,11 @@ void Spaceship::onFixedUpdate()
                 rope.destroy();
             }
         }
+        if (controls->restart.getDown() && controls->restart.get())
```

Comment:
copied get and getDown from previous usage but I guess that might have been there to assure both buttons were pressed simultaneously?

If so I can leave out the .get()

Reply:
getDown() is active for 1 cycle, while get() is active as long as the button is down.

I had to use both, as "getDown()" on 2 buttons in the same cycle wouldn't really work. And only checking get() would result in resetting the game as long as the button is pressed. In this case, getDown is enough. As you only want to have the "button down" event.